.travis.yml

```diff
@@ -106,6 +106,12 @@ matrix:
               false;
           fi
 
+    - <<: *basic-vm
+      name: "UTF-8 Check"
+      script:
+        # Make sure we're not introducing any non-UTF-8 character in text files
```

Comment:
This comment doesn't make sense. Pretty much any character is going to be encodable by UTF-8.

Reply:
I can rephrase - meant invalid encoding